pkg/controller/controller.go

```diff
@@ -98,17 +112,38 @@ func NewResizeController(
 		DeleteFunc: ctrl.deletePVC,
 	}, resyncPeriod)
 
+	podInformer.Informer().AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
```

Comment:
I see that the in-tree PVC protection controller also checks the pod.  What is the advantage of checking thru the pod vs checking the VolumeAttachment object?

Reply:
Because not all volume types support attach/detach and hence there may not be an VolumeAttachment object.